frontend/app/src/components/Sidebar/SidebarNav.tsx

```diff
@@ -49,6 +49,9 @@ export interface Props {
   onPageChange: (pageName: string) => void
 }
 
+const COLLAPSE_THRESHOLD = 13
+const NUM_PAGES_TO_SHOW_WHEN_COLLAPSED = 10
```

Comment:
Think comment(s) would be helpful here to make distinction more digestible without spec context